src/pages/api/instance.ts

```diff
@@ -0,0 +1,12 @@
+import axios from 'axios';
+
+const axiosInstance = axios.create({
+  baseURL: 'https://api.wisey.app/api/v1',
+  headers: {
+    'Content-Type': 'application/json',
+    Authorization:
+      'Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIwMDAwZDE4My0yZTk1LTRlYjktOWQ1ZS0zODA2NTUyNjFhNWUiLCJwbGF0Zm9ybSI6InN1YnNjcmlwdGlvbnMiLCJpYXQiOjE2Nzg5OTgzNDQsImV4cCI6MTY3OTg5ODM0NH0.G2cQQVKC9VgP3SbQ7f15vTqnabcfw-2x0O2m0bAWdv0',
```

Comment:
A hard-coded token is a bad practice. You have an API path in Postman that you can use to get the token. Otherwise, it's better to put the token in the .env.local file and get it as, for example, ```process.env.API_TOKEN```